src/app.test.ts

```diff
@@ -1,51 +1,202 @@
-import { test, beforeEach, afterEach } from 'tap'
+import { afterEach, before, beforeEach, test } from 'tap'
 import tmp from 'tmp'
+import path from 'path'
+import fs from 'fs-extra'
+import { FastifyInstance } from 'fastify'
 
 import app from './app'
 import mapboxRasterTilejson from './fixtures/good-tilejson/mapbox_raster_tilejson.json'
-import { getTilesetId } from './lib/tilestore'
+import { getTilesetId } from './lib/utils'
+import { TileJSON, validateTileJSON } from './lib/tilejson'
 
 tmp.setGracefulCleanup()
 
-beforeEach((done, t) => {
+type TestContext = {
+  server: FastifyInstance
+  sampleTileJSON: TileJSON
+}
+
+before(() => {
+  function assertSampleTileJSONIsValid(
+    data: unknown
+  ): asserts data is TileJSON {
+    if (!validateTileJSON(data)) {
+      const message = `Sample input does not conform to TileJSON schema spec: ${JSON.stringify(
+        validateTileJSON.errors,
+        null,
+        2
+      )}`
+
+      throw new Error(message)
+    }
+  }
+
+  assertSampleTileJSONIsValid(mapboxRasterTilejson)
+})
```

Comment:
this is mostly to ensure that whatever fixture we use conforms to the desired schema structure. allows us to change the fixture with more confidence in the future if needed